front/databaseinstance.form.php

```diff
@@ -39,6 +39,14 @@
 
 Session::checkRight('database', READ);
 
+Html::header(
+    DatabaseInstance::getTypeName(Session::getPluralNumber()),
+    $_SERVER['PHP_SELF'],
+    "management",
+    "database",
+    "databaseinstance"
+);
```

Comment:
It should probably be added only in the last `} else {` block.

Reply:
Change has been applied, thank.